File: backend/Entities/Models/Bairro.cs
using System;
using System.Collections.Generic;

namespace Entities.Models;

public partial class Bairro
{
    public int Id { get; set; }

    public string Nome { get; set; } = null!;

    public int MunicipioId { get; set; }

    public DateTime? DataCadastro { get; set; }

    public virtual ICollection<FilialLastMile> FilialLastMiles { get; } = new List<FilialLastMile>();
}

File: frontend-with-menus/src/app/Business/Models/bairro.ts
import { FilialLastMile } from "./FilialLastMile";
import { Municipio } from "./Municipio";
import { Estado } from "./estado";

export interface Bairro   {
    id: number;
    nome: string;
    municipioId: number;
    filialLastMiles: FilialLastMile[];
    municipio: Municipio;
}
